scripts/coverage.sh

```diff
@@ -54,12 +52,17 @@ TEST_ARGS=(
   --skip shred::merkle::test::test_recover_merkle_shreds::
 )
 
-cargo +"$rust_nightly" test --target-dir "$here/../target/cov" "${PACKAGES[@]}" -- "${TEST_ARGS[@]}"
+# Most verbose log level (trace) is enabled for all solana code to make log!
+# macro code green always. Also, forcibly discard the vast amount of log by
+# redirecting the stderr altogether on CI, where all tests are run unlike
+# developing.
+RUST_LOG="solana=trace,agave=trace,$RUST_LOG" INTERCEPT_OUTPUT=/dev/null "$here/../ci/intercept.sh" \
+  cargo +"$rust_nightly" test --target-dir "$here/../target/cov" "${PACKAGES[@]}" -- "${TEST_ARGS[@]}"
```

Comment:
hope no more overflow test failures in 1 hour.

Reply:
> hope no more overflow test failures in 1 hour.

seems we did it: https://buildkite.com/anza/agave/builds/4511#018f74f3-4da8-4cb2-aa63-d6c7d49d2f76